lib/users/api.js

```diff
@@ -576,7 +571,7 @@ class API {
    * @throws {APITimeoutError} if the maximum wait time was exceeded
    * @throws {APIError} if an underlying API request was unsuccessful
    */
-  async waitUntilJobCompletes(jobId, sleepTime=5, maxWaitTime=600) {
+  waitUntilJobCompletes(jobId, sleepTime=5, maxWaitTime=600) {
```

Comment:
did you test?

Reply:
This is incorrect - I fixed it in my `batch-actions` branch

Reply:
Yep

Reply:
> This is incorrect - I fixed it in my `batch-actions` branch

What's the issue? Seems to work fine for me

Reply:
Didn't fully explain on mobile, but for reference - you're right, it would work without the `async`. However, because it returns a promise and is intended to be called with `await`, keeping the `async` is more informative (and to keep the linter happy, you need an `await` inside the function).